src/frontend/src/views/Upload/components/Metadata/components/ImportFile/style.ts

```diff
@@ -79,3 +87,21 @@ export const StyledButton = styled(Button)`
     `;
   }}
 `;
+
+export const VerticalLine = styled.div`
+  height: 100%;
+  width: 1px;
+  margin: 0;
+
+  &:before {
+    content: "\\00a0";
```

Comment:
what's going on here?

Reply:
This is a non-breaking space, same as `&nbsp;`. The div won't render without it.

Reply:
the reason the div doesn't render is because you have height set to 100%, but that doesn't have an effect if the height isn't explicitly set on the parent element. So `height: 100%` takes no effect, giving you a div with `height: 0`.  If you define the height as some specific pixel value, then it will appear without having to add the space.

https://github.com/chanzuckerberg/czgenepi/pull/1451/commits/df7abbc9d98fa3c21add1a434bb8ae6f4cb330a5